test/blackbox-tests/test-cases/pkg/ocamlformat/binary-avoid-conflict-with-project.t

```diff
@@ -0,0 +1,110 @@
+Make sure that the "dune-project" does not use the OCamlFormat binary dev-tool.
+
+  $ . ../helpers.sh
+  $ mkrepo
+
+Make a fake ocamlformat:
```

Comment:
The logic for making a fake ocamlformat is duplicated in most of these tests. Can you move it to a single file which is sourced in each test like we do with `helpers.sh`.